kernel/packages/unity-interface/BrowserInterface.ts

```diff
@@ -465,7 +470,11 @@ export class BrowserInterface {
   }
 
   public KillPortableExperience(data: { portableExperienceId: string }) {
-    stopPortableExperienceWorker(data.portableExperienceId)
```

Comment:
This code is duplicated from `dcl.ts` as I can't import the method here as it's a circular reference.